Tests/AsyncHTTPClientTests/HTTPClientTests.swift

```diff
@@ -632,7 +637,10 @@ class HTTPClientTests: XCTestCase {
     }
 
     func testResponseFutureIsOnCorrectEL() throws {
-        let group = MultiThreadedEventLoopGroup(numberOfThreads: 4)
+        let group = getDefaultEventLoopGroup(numberOfThreads: 4)
+        defer {
+            XCTAssertNoThrow(try group.syncShutdownGracefully())
```

Comment:
shouldn't this be executed by the tests teardown?

Reply:
In this case the eventLoopGroup has been created inside the test because it required a different number of threads